avalon/harmony/lib.py

```diff
@@ -109,7 +109,7 @@ def launch_zip_file(filepath):
     # Launch Avalon server.
     self.server = Server(self.port)
     thread = threading.Thread(target=self.server.start)
-    thread.deamon = True
+    thread.daemon = True
```

Comment:
This is not a mispelling; https://docs.python.org/3/library/threading.html#threading.Thread.daemon

Please revert this.

Reply:
Are you sure? It was spelt deamon but as the doc says that you posted it should be daemon.

Reply:
Sorry my bad.